src/v2/Apps/ArtistSeries/Components/ArtistSeriesHeader.tsx

```diff
@@ -24,59 +84,82 @@ const ArtistSeriesHeader: React.FC<ArtistSeriesHeaderProps> = props => {
 
 const ArtistSeriesHeaderLarge: React.FC<ArtistSeriesHeaderProps> = props => {
   const {
-    artistSeries: { title, description },
+    artistSeries: { title, description, artists },
   } = props
   return (
-    <Box>
-      <Grid>
-        <Row>
-          <Col sm={6}>
-            <Flex
-              height="100%"
-              flexDirection="column"
-              justifyContent="space-between"
-            >
-              <Sans size="8" element="h1" unstable_trackIn>
-                {title}
-              </Sans>
-              <Sans lineHeight={1.5} size="3t" pr={1}>
-                {description}
-              </Sans>
-            </Flex>
-          </Col>
-          <Col sm={6}>
-            <Box
-              // FIXME: Use Image
-              bg="black10"
-              width={1}
-              height={400}
-            ></Box>
-          </Col>
-        </Row>
-      </Grid>
-    </Box>
+    <>
+      <Flex
```

Comment:
I wonder if we should revisit the designs with Nicole about this Grid layout and instead rely on Flexbox to handle the centering and layout. We only use the Grid in a few places (e.g. some editorial layouts, the Artwork Grid) and I'm not sure it makes sense to do it here. It might, but it might be worth revisiting with her.

Reply:
Also I'm curious is this is why you had to use position relative and absolute to override the grid layout, in which case might be another reason to use Flexbox.

Reply:
@ashleyjelks Did you mean to make this comment here? This uses `Flex` components though we use a `Grid` below.

Using Flexbox had some issues in getting the right layout in the "Series" header -- we want `ArtistInfo` on the left and `Series` always centered. I don't know of a way to do that in Flexbox without a third component on the right. That led me here: https://stackoverflow.com/a/38948646, which is how I ended up using `position: absolute`.